docs/docs/how-to/profile_selectors.md

```diff
@@ -0,0 +1,75 @@
+  - entity: artifact
+    selector: artifact.provider.name == 'github-app-stacklok'
+```
+
+Let's break down the example above:
+- `entity`: Defines the type of entity you want to filter (`repository`, `artifact`, or `pull_request`).
+- `selector`: The CEL expression that specifies the filtering criteria. In the example:
+  - The first selector filters repositories to include only those that are not forks and whose name starts with `stacklok/`, in other words those that are part of the stacklok organization.
+  - The second selector filters artifacts to include only those provided by `github-app-stacklok`.
+
+Below you can find the full list of selectors available for each entity type.
+
+## Repository selectors
+
+Selectors for repositories allow you to filter and manage repositories based on specific attributes.
+
+| Field        | Description                                                                                         | Type             |
+|--------------|-----------------------------------------------------------------------------------------------------|------------------|
+| `name`       | The full name of the repository, e.g. stacklok/minder                                               | string           |
+| `is_fork`    | `true` if the repository is a fork, `nil` if unknown or not applicable to this provider             | bool             |
+| `is_private` | `true` if the repository is private, `nil` if unknown or not applicable to this provider            | bool             |
+| `provider`   | The provider of the repository, for me details see [Provider Selectors](#entity-provider-selectors) | ProviderSelector |
+
+## Artifact selectors
+
+| Field      | Description                                                                                       | Type             |
+|------------|---------------------------------------------------------------------------------------------------|------------------|
+| `name`     | The full name of the artifact, e.g. stacklok/minder-server                                        | string           |
+| `type`     | The type of the artifact, e.g. "container"                                                        | string           |
+| `provider` | The provider of the artifact, for me details see [Provider Selectors](#entity-provider-selectors) | ProviderSelector |
+
+## Pull request selectors
+
+| Field  | Description                                                 | Type   |
+|--------|-------------------------------------------------------------|--------|
+| `name` | The full name of the pull request, e.g. stacklok/minder/123 | string |
+
+## Entity provider selectors
+
+Each entity can be filtered based on its provider.
+
+| Field   | Description                                        | Type   |
+|---------|----------------------------------------------------|--------|
+| `name`  | the name of the provider, e.g. github-app-stacklok | string |
```

Comment:
nitpick: Capitalize the first word of the description to be consistent with the rest of the table. 
**The** name of the provider,